dlp/snippets/deid.py

```diff
@@ -937,6 +937,76 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_simple_word_list]
+def deidentify_with_simple_word_list(
+    project: str,
+    input_str: str,
+    custom_info_type_name: str,
+    word_list: list,
```

Comment:
What is it a list of? list of String?

Reply:
Be specific with the type of list (you need to import from `typing` to annotate types inside):
```
from typing import List
    word_list: List[str],
```

starting 3.8 you should be able to use built-ins rather than from the `Typing` library:
```
    word_list: list[str],
```

starting 3.9 we'd recommend being specific with the type of list:
```
from collections.abc import Sequence
    word_list: Sequence[str],
```

Reply:
okay got it, Updated , please check. Thank you for all the suggestions